pages/ai-ecosystem/integrations.mdx

```diff
@@ -533,4 +535,226 @@ answer.
 
 ![langchain-query](/pages/ai-ecosystem/integrations/langchain-retrieval.png)
 
+
+## Cognee
+
+<Cards>
+  <Cards.Card
+    icon={<GitHub />}
+    title="Cognee integration"
+    href="https://github.com/topoteretes/cognee/tree/main/cognee/infrastructure/databases/graph/memgraph"
+  />
+</Cards>
```

Comment:
This needs to be external link - I think we have a component for that, let me know if you can't find it (check cards from the community links at the bottom)